### PR TITLE
activestorage - IE11 direct_upload error

### PR DESCRIPTION
**Pull request by kylekeesling**
### Summary 
Direct upload was throwing an error after the assets were uploaded to a service, specifically when attempting to attach a submit button to the form and click to send the form.

More background and info is outlined in #32378.

### CODE REVIEW

File: activestorage/app/javascript/activestorage/ujs.js [hunk 1]
```diff
@@ -58,8 +58,8 @@ function submitForm(form) {
     button.disabled = disabled
   } else {
     button = document.createElement("input")
-    button.type = "submit"
-    button.style = "display:none"
+    button.setAttribute("type", "submit")
```
Comment (javan) [inline activestorage/app/javascript/activestorage/ujs.js]: Is this change required too? Looking at #32378 and other search results for "Assignment to read-only properties is not allowed in strict mode" it seems like the issue is just with setting `button.style = "…"`.

Comment (kylekeesling) [reply activestorage/app/javascript/activestorage/ujs.js]: From my testing it required both to be changed. Once I changed the style attribute I got the same error on the way that the type was set to submit. Making these two changes then allowed it to work properly.

Comment (kylekeesling) [reply activestorage/app/javascript/activestorage/ujs.js]: I spelled that out on a comment below but I’m happy to update the issue description if that helps as well.

Comment (javan) [reply activestorage/app/javascript/activestorage/ujs.js]: Hmm, I'm only seeing an error in IE 11 setting `button.style = "…"` (in strict mode only). Tested with:
```js
(function() {
  "use strict";
  
  var button = document.createElement("button")
  
  try {
    button.type = "submit"
  } catch (error) {
    console.log("error setting button.type", error)
  }

  try {
    button.style = "display:none"
  } catch (error) {
    console.log("error setting button.style", error)
  }

  try {
    button.style.display = "none"
  } catch (error) {
    console.log("error setting button.style.display", error)
  }
})()
```

Comment (kylekeesling) [reply activestorage/app/javascript/activestorage/ujs.js]: You're correct - when I first began testing I wasn't sure how to run off of a local rails gem in my gemfile, so when I ran my testing I may not have properly testing against just the style change.

I've just confirmed on my side that simply changing the style configuration step solves the problem.

Sorry for the confusion on this!

Comment (kylekeesling) [reply activestorage/app/javascript/activestorage/ujs.js]: Would you like me to submit a new PR? Happy to do so if you'd find it useful.

**Comment by kylekeesling** [comment]
I also noticed that I opened this PR against master rather than the 5.2.0 branch. I've got a version ready to go that makes this change on that branch instead. Let me know if you'd rather close this PR out and open another one using the branch 5.2.0 branch.

**Comment by javan** [comment]
Generally, you should always open PRs against master. We'll backport the commits as necessary. You can either [rebase and squash your commits](http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits) in this PR. Or, just open a new one if that's easier.

**Comment by kylekeesling** [comment]
Ill just open a new PR with the change
